packages/api-reference/src/components/Content/Models/ModelTag.vue

```diff
@@ -34,14 +29,10 @@ const headerId = useId()
       v-if="options.layout === 'modern'"
       id="models">
       <Section
-        :aria-labelledby="headerId"
+        aria-label="Models"
```

Comment:
We don't really need to use an id here since it's always the same label. Slightly fewer dom attribute this way